playbooks/roles/deploy-osh/tasks/main.yml

```diff
@@ -9,7 +9,8 @@
     - always
 
 - name: Get ses_cluster_configuration
-  include_vars: "{{ socok8s_ses_pools_details }}"
```

Comment:
Could we keep `include_vars:` I think it would be advisable instead of filtering.

The code would be:
```
include_vars:
  file: "{{ socok8s_ses_pools_details }}"
  name: ses_cluster_configuration